### PR TITLE
some style change and zh-CN localization

### PR DESCRIPTION
**Pull request by tiansh**
* changed the layout of proxy list, moved buttons and selects to the right
* Add i18n support
* Add Chinese (China) localization

Issue: #10

### CODE REVIEW

**Comment by bekh6ex** [comment]
@tiansh Thank you very much for your contribution! I'm quite excited about that.

I have little time to work on it. Can you fix the merge conflicts?
It should mostly be formatting. If not, I can get back to it later.

**Comment by tiansh** [comment]
It seems that the conflicts are caused by your reformatting. Which tools / commands you are using to format your codes? If it only format issue and nothing functional changed, I believe merging force and run the format tool again would be a better choice.

File: src/options/ProxyList.js [hunk 1]
```diff
@@ -28,27 +28,30 @@ export class ProxyList {
 
   view () {
     const items = this.model.list.map(this.renderProxyItem.bind(this))
-    const actions = m('div', [
-      m('a[href=/proxies/new]', { oncreate: m.route.link, class: style.primaryButton }, '+')
+    const actions = m('div.proxy-list-item', [
+      m('.proxy-name'),
```
Comment (bekh6ex) [inline src/options/ProxyList.js]: seems like left here by accident.

File: src/options/options.css [hunk 2]
```diff
@@ -128,22 +133,13 @@
 }
 
 header {
-    position: absolute;
-    left:0;
-    right: 0;
-    top:0;
```
Comment (bekh6ex) [inline src/options/options.css]: need to check

Comment (bekh6ex) [reply src/options/options.css]: Ok. I got it.
Actually, the original intention was to have a fixed layout with a scroll bar in the list area, but apparently, I didn't do it 😛.
I will fix it later. Thank you for cleanup.

File: src/options/ProxyList.js (hunk 1 above)
Comment (bekh6ex) [reply src/options/ProxyList.js]: As I understood it is to have the same look. 
I did some CSS copypaste, but made it have a distinct class.

**Comment by bekh6ex** [comment]
Thank you for the contribution! 👍

**Pull request merged**